cmake/custom/compilers/Clang.CXX.cmake

```diff
@@ -8,7 +8,7 @@ if(NOT DEFINED ENV{CXXFLAGS})
     endif()
 
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${CXX_STANDARD_FLAG} -Qunused-arguments -fcolor-diagnostics")
-    set(CMAKE_CXX_FLAGS_DEBUG    "-O0 -DDEBUG -Wall -Wextra -Winit-self -Woverloaded-virtual -Wuninitialized -Wmissing-declarations -Wwrite-strings -Weffc++ -Wdocumentation -Wno-sign-compare")
+    set(CMAKE_CXX_FLAGS_DEBUG    "-O0 -DDEBUG -Wall -Wextra -Winit-self -Woverloaded-virtual -Wuninitialized -Wmissing-declarations -Wwrite-strings -Weffc++ -Wno-sign-compare")
```

Comment:
This flag should be kept. It signals problems with documentation syntax.

Reply:
Fixed with commit b7816c32